templates/index.html

```diff
@@ -1,18 +1,29 @@
 {% extends "layout/_template.njk" %}
-{%
-  set pageConfig = {
-      "header": {
-          "title": "Service name"
-      },
-      "footer": {
-          "OGLLink": {
-              "pre": "All content is available under the", "link": "Open Government Licence v3.0",
-              "url": "https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
-              "post": ", except where otherwise stated"
-          }
-      }
-  }
-%}
+{% from "components/button/_macro.njk" import onsButton %}
 {% block main %}
-  <h1>{{ param }}</h1>
-{% endblock %}
+  {% with messages = get_flashed_messages() %}
+  {% if messages %}
+    <ul class="flashes">
+      {% for message in messages %}
+        <li>{{ message }}</li>
+      {% endfor %}
+    </ul>
+  {% endif %}
+  {% endwith %}
+  <section id="section-content" class="ons-page__body">
+    <h2>What is the Tech Radar Data Collection Tool?</h2>
+    <p>
+      This tool is used to collect data about various projects, which is then utilized by the Tech Radar Tool. This helps in
+      assessing and visualizing the technology landscape within the organization, aiding in strategic decision-making and
+      technology adoption.
+    </p>
+    <p>
+      Selected users can sign in to the tool using their Cognito credentials. Once signed in, they can create a project and
+      view existing project data.
+    </p>
+    {{ onsButton({
+            "text": "Sign in with Cognito",
+            "url": "https://keh-tech-audit-tool.auth.eu-west-2.amazoncognito.com/oauth2/authorize?client_id=dm3289s0tqtsr5qn2qm5i9fql&response_type=code&scope=email+openid+phone&redirect_uri=http%3A%2F%2Flocalhost:8000"
```

Comment:
Can we extract the client_id and redirect_uri to variables please. It is best practice to avoid showing the client_id as a hardcoded string, we should read this from secret manager.

Reply:
Done